### PR TITLE
spark.rst: Add standalone-Spark use case tutorial

### PR DESCRIPTION
**Pull request by joselamego**
This tutorial walks the user through the process of installing, configuring, and running Apache* Spark on Clear Linux OS for Intel(r) Architecture.

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>

### CODE REVIEW

**Comment by MCamp859** [comment]
Sorry, new user error.

**Comment by joselamego** [comment]
@MCamp859 : Thanks for the review. Is there anything I must do now as next step?

**Comment by MCamp859** [comment]
@joselamego, I think @rcaballeromx plans to review my edits and then he is the final approver. 
If I am wrong, he will let us know.

**Comment by rcaballeromx** [comment]
@MCamp859 I don't see a commit with your changes. Where there none?

**Comment by MCamp859** [comment]
@rcaballeromx @joselamego  looks like my commit is in branch "spark1"  [my error when creating remote] https://github.com/joselamego/clear-linux-documentation/tree/spark1 
Sorry for the hassle. Please let me know how to fix this.

**Comment by MCamp859** [comment]
@rcaballeromx @joselamego Commit is pushed to this branch.

**Comment by rcaballeromx** [comment]
@joselamego Can you take a look at the changes @MCamp859 did and approve or request changes? Thanks!

**Comment by joselamego** [comment]
I'm ok with @MCamp859 's changes. Thanks guys.

File: source/clear-linux/tutorials/spark.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+#. Copy the template files below to create custom configuration files:
+
+   .. code-block:: bash
+
+      sudo cp /etc/spark/spark-defaults.conf.template /etc/spark/spark-defaults.conf
+      sudo cp /etc/spark/spark-env.sh.template /etc/spark/spark-env.sh
+      sudo cp /etc/spark/log4j.properties.template /etc/spark/log4j.properties
+
+#. Edit the :file:`/etc/spark/spark-env.sh` file and add the
+   :option:`SPARK_MASTER_HOST` variable. Replace the example address below
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: This is not an option, please use the \:envvar\: role instead

File: source/clear-linux/tutorials/spark.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+   with your localhost IP address. (View your IP address using the
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Do not use parenthesis. Remove them or make the line a note. The use of parenthesis is ambiguous and should be avoided.

File: source/clear-linux/tutorials/spark.rst [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+   .. code-block:: xml
+
+      SPARK_MASTER_HOST="10.300.200.100"
+
+   .. note:: This optional step enables the master's web user interface to
+      view information needed later in this tutorial.
+
+#. Edit the :file:`/etc/spark/spark-defaults.conf` file and update the
+   :option:`spark.master` variable with the :option:`SPARK_MASTER_HOST` address
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Do not use the option role for everything. Use the appropriate roles as found here: http://www.sphinx-doc.org/en/master/markup/inline.html
If no role is suited, simply use back-ticks \`\`

File: source/clear-linux/tutorials/spark.rst [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+   from the previous step and port :option:`7077`.
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Remove \:option\:. See previous comments regarding role usage.

File: source/clear-linux/tutorials/spark.rst [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+   .. code-block:: xml
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: This code does not look like XML code. If the language cannot be determined leave the language blank. @joselamego can you confirm the language used in this code-block?

File: source/clear-linux/tutorials/spark.rst [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+*******************************************
+
+#. Start the master server using:
+
+   .. code-block:: bash
+
+      sudo /usr/share/apache-spark/sbin/./start-master.sh
+
+#. Start one worker daemon and connect it to the master using the
+   :option:`spark.master` variable defined earlier:
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Remove \:option\:

File: source/clear-linux/tutorials/spark.rst [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+   .. code-block:: bash
+
+      sudo /usr/share/apache-spark/sbin/./start-slave.sh spark://10.300.200.100:7077
+
+#. Open an internet browser and view the worker daemon information using
+   the master's IP address and port :option:`8080`:
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Remove \:option\:

File: source/clear-linux/tutorials/spark.rst [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+   .. code-block:: xml
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Remove xml

File: source/clear-linux/tutorials/spark.rst [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+#. Run the wordcount example using a file on your local host and output the
+   results to a new file with the following command:
+
+   .. code-block:: bash
+
+      sudo spark-submit /usr/share/apache-spark/examples/src/main/python/wordcount.py ~/Documents/example_file > ~/Documents/results
+
+#. Open an internet browser and view the application information using
+   the master's IP address and port :option:`8080`:
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Remove \:option\:

File: source/clear-linux/tutorials/spark.rst [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+   .. code-block:: xml
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: remove xml

File: source/clear-linux/tutorials/spark.rst [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+      http://10.300.200.100:8080
+
+#. View the results of the wordcount application in the results file specified
```
Comment (rcaballeromx) [inline source/clear-linux/tutorials/spark.rst]: Do not reference a previous step. State explicitly the name and path of the file.

File: source/clear-linux/tutorials/spark.rst (hunk 5 above)
Comment (joselamego) [reply source/clear-linux/tutorials/spark.rst]: It is standard shell scripting. Leaving language blank seems ok.

**Comment by rcaballeromx** [comment]
@AthenasJimenez @joselamego Can you make sure the content is still technically accurate? Once you approve, we can merge the content to publish it.

**Comment by joselamego** [comment]
Ok with me. Thanks

**Pull request merged**